### ISSUE TITLE
Process crashed when parsing numbers

### ISSUE THREAD

**Issue by mesin** [axa-group/nlp.js#200]
**Describe the bug**
Recognition of text raise an error when I have trained data on Hebrew and enter any number

**To Reproduce**
1. Create Hebrew data
2. Compile and write only a digit
3. See error

**Desktop (please complete the following information):**
 - OS: MacOS
 - nlp.js - lastest master

**Additional content**

```
bot> מנהל המערכת יתקשר אליך. תודה.
1
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Function.filterLanguages (/Users/meshin/tmp/nlp.js/lib/language/language.js:146:12)
    at Function.getDistances (/Users/meshin/tmp/nlp.js/lib/language/language.js:161:32)
    at Function.detectAll (/Users/meshin/tmp/nlp.js/lib/language/language.js:191:32)
    at Language.guess (/Users/meshin/tmp/nlp.js/lib/language/language.js:250:29)
    at NluManager.guessLanguage (/Users/meshin/tmp/nlp.js/lib/nlu/nlu-manager.js:81:32)
    at NluManager.fillLanguage (/Users/meshin/tmp/nlp.js/lib/nlu/nlu-manager.js:230:28)
    at NluManager.getClassifications (/Users/meshin/tmp/nlp.js/lib/nlu/nlu-manager.js:269:10)
    at NlpManager.process (/Users/meshin/tmp/nlp.js/lib/nlp/nlp-manager.js:363:34)
    at Interface.rl.on (/Users/meshin/tmp/nlp.js/examples/console-bot/index.js:35:41)
```

<img width="1424" alt="Снимок экрана 2019-06-21 в 14 54 50" src="https://user-images.githubusercontent.com/297536/59920870-96c1bc00-9434-11e9-9149-1af667418576.png">

**Comment by mesin** [comment]
Hey guys

Can anyone please help to understand how can I fix it?
Should I bring some changes to Hebrew language?

I would like to help if I would know what to do. 
It is integrated in twilio custom bot which should be able to talk with customers on their native languages.

**Comment by jesus-seijas-sp** [comment]
Hi! 
The problem was the language identifier. It at process you provide the language, then will be not problem.

Also, the problem is fixed, and published as version 3.2.0

**Comment by mesin** [comment]
Thanks for fix! It's working
